tests/test_xfail.py

```
@@ -0,0 +1,35 @@
+    return
+
+
+@pytest.mark.xfail(
+    reason=(
+        "Network infrastructure is slow from time to time. "
+        "Due to that, test might fail with packet loss greater than 2%"
+    ),
+    raises=HighPacketLossError,
+)
+@pytest.mark.polarion("CNV-113")
+def test_3():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)